.gitignore

```diff
@@ -63,3 +63,5 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots
 fastlane/test_output
+.DS_Store
+/JordanResources
```

Comment:
maybe put them somewhere else on your machine or an your personal gitignore? not sure we would want something like this in our gitignore file otherwise it gets messy :)